libs/ui/editor/src/lib/atoms/addBlockInput.tsx

```diff
@@ -26,6 +26,7 @@ export interface AddBlockInputProps {
 
 const Wrapper = styled.div`
   position: relative;
+  left: 41.5%;
```

Comment:
Whats this achieving?

Reply:
basically Michael wanted to align the + button more or less with the box that renders the teaser. I know the solution is not perfect, but I don't think that it really makes sense and still, after my fix it looks better than before

https://wepublish.atlassian.net/jira/software/c/projects/WP/boards/25?search=158&selectedIssue=WP-158

<img width="1013" alt="Screenshot 2024-04-17 at 14 55 00" src="https://github.com/wepublish/wepublish/assets/24550916/8b6e59d4-630f-4066-8d5f-6dfd69de9eff">